src/layer/hillshade.js

```diff
@@ -0,0 +1,19 @@
+export const hillshading = {
+  id: "hillshading",
+  type: "hillshade",
+  source: "dem",
+  paint: {
+    "hillshade-exaggeration": [
+      "interpolate",
+      ["linear"],
+      ["zoom"],
+      3,
+      0.2,
+      12,
+      0.5,
+    ],
+    "hillshade-shadow-color": "rgba(102,85,51,1)",
+    "hillshade-highlight-color": "rgba(255,255,204,1)",
+    "hillshade-accent-color": "rgba(0,0,0,1)",
```

Comment:
With these colors, some road lines don’t have a lot of contrast against the background, while the knockouts around labels stand out enough to look like halos again. Maybe we could try moving all three colors closer to the preexisting background color. This could also help the layer recede into the background while still conveying information about rugged terrain to those who look for it.

Reply:
Tamed it a _lot_. Since the hillshade is now restricted to the luminance space, I increase the exaggeration at low zooms since it no longer fights with other colors. It fades away at high zooms to avoid ruining the foreground by being a murky blob, and to avoid getting confused by things like stadiums.

Reply:
In the flat lands, the old version made it actually look like a crumpled paper map :-D